python/nammu/SOAPClient/SOAPClient.py

```diff
@@ -58,3 +84,41 @@ def create_request_zip(self):
         Pack attachment in a zip file.
         """
         pass
+
+    def get_server_logs(self):
+        """
+        Manipulate response to substract the content of oracc.log that is in the
+        returned binary-coded zip file.
+        """
+        self.response.content
+        binary_body = re.split('--==.*==', self.response.content)[2].split('\r\n')[5]
+
+        f = StringIO.StringIO()
+        f.write(bytearray(binary_body))
+
+        # if zipfile.is_zipfile(f):
+        memory_zip = ZipFile(f)
+        zip_content = {name: memory_zip.read(name) for name in memory_zip.namelist()}
+        oracc_log = zip_content['oracc.log']
+        request_log = zip_content['request.log']
+        # See if server returns a lemmatised file
+        autolem = None
+        for key, value in zip_content.iteritems():
+            if key.endswith("autolem.atf"):
+                autolem = zip_content[key]
+
+        print "@"*30
+        print oracc_log
+        print "@"*30
+        print request_log
+        print "@"*30
+        if autolem:
+            print autolem
+            print "@"*30
+
+        return oracc_log, request_log, autolem
+
+    def get_autolem(self):
```

Comment:
This function should either do something, be removed or at lease have a pass in the body